winterfell/webapp/src/js/app.js

```diff
@@ -4,56 +4,126 @@
 'use strict';
 
 var app = angular.module('vetafiApp', [
-  'ngRoute',
   'angular-click-outside',
-  'ngDialog',
   'signature',
   'formly',
   'formlyBootstrap',
   'angular-momentjs',
-  'ui.bootstrap'
+  'ui.bootstrap',
+  'ui.router'
 ]);
 
 /**
  * Configure routes
  */
-app.config(function ($routeProvider) {
-  $routeProvider.when("/", {
-    templateUrl: "templates/home.html"
-  }).when("/faq", {
-    templateUrl: "templates/faq.html"
-  })
-
-    // Profile Pages
-    .when("/profile", {
-      templateUrl: "templates/profile.html"
-    }).when("/profile/:subPage", {
-      templateUrl: "templates/profile.html"
-    })
-
-    // File Claims Pages
-    .when("/tos", {
-      templateUrl: "templates/tos.html"
-    }).when("/claim/start", {
-      templateUrl: "templates/claimStart.html"
-    }).when("/claim/:claimId/select-forms", {
-      templateUrl: "templates/claimSelectForms.html"
-    }).when("/claim/confirm", {
-      templateUrl: "templates/claimConfirm.html"
-    }).when("/claim/submitted", {
-      templateUrl: "templates/claimSubmitted.html"
-    }).when("claim/form/:formId", {
-      template: "<div>Some Form</div>" // replace to claim/form/:formId when it's ready
-    }).when("/claim/:claimId", {
-      templateUrl: "templates/claimView.html"
-    })
-
-    .when("/claim/:claimId/form/:formId", {
-      templateUrl: "templates/form.html"
-    })
-
-    // Otherwise
-    .otherwise({
-      redirectTo: '/'
-    });
-});
+app.config(['$stateProvider', '$urlRouterProvider', function ($stateProvider, $urlRouterProvider) {
+
+  $stateProvider.state({
+    name: 'root',
+    url: '/app',
+    abstract: true,
+    templateUrl: 'templates/root.html',
+    controller: 'headerCtrl',
+    resolve: {
+      user: ['Profile', function(Profile) {
+        return Profile.resolveUser();
+      }]
+    }
+  });
+
+  $urlRouterProvider.otherwise('/app');
+
+  // Home
+  $stateProvider.state({
+    name: 'root.home',
+    url: '',
```

Comment:
By leaving the url empty here, `root.home` becomes the default state for the path `/app`